### DISCUSSION TITLE
How to use external variables in argocd?

### DISCUSSION THREAD

**Discussion by artSerenade** [argoproj/argo-cd#10345] [Q&A]
I set my external variables like this:
![image](https://user-images.githubusercontent.com/45325326/184812066-ba22ba92-e27d-4e8c-b404-c85b2b94d8d7.png)
I use this external variables in deployment like this:
![image](https://user-images.githubusercontent.com/45325326/184812550-97e169ec-b8fb-4f3f-8042-c5b7f4616545.png)
but it doesn't work in argocd:
![image](https://user-images.githubusercontent.com/45325326/184813073-66106900-f995-4639-9de5-8a991846d181.png)
What should I do?

**Comment by pasha-codefresh** [comment]
The directory type it is jsonnet 

https://argo-cd.readthedocs.io/en/stable/user-guide/jsonnet/

you can see all details here and info about external variables
